tutorial/part1/chapter2.md

```diff
@@ -1,78 +1,76 @@
 # Chapter 2: Getting Started with Dash
 
 ## What you will learn
-This chapter sets the foundation for the creation of Dash Applications. Starting from a minimal example, we'll explain the structure of a Dash app, how to interact with it, and how to update it. By the end of the chapter, you'll understand the following code and know how to launch your first Dash app:
+This chapter sets the foundation for the creation of Dash Applications. Starting from a minimal example, we'll explain the structure of a Dash app, how to interact with it, and how to update it. By the end of the chapter, you'll understand the following code of a minimal dash app, and know how you can launch it as your first Dash app:
```

Comment:
I personally prefer it less worthy, Joel. However, I'm good either way. If you think adding the fact that it is a minimal Dash app makes a big difference, feel free to keep your change. (Just capitalize D in Dash)

Reply:
Updated to be less wordy but retain most of the sentiment I was after.